src/renderer/widgets/IWidget.hpp

```diff
@@ -23,5 +25,5 @@ class IWidget {
         bool        allowForceUpdate = false;
     };
 
-    virtual SFormatResult formatString(std::string in);
+    static SFormatResult formatString(std::string in);
```

Comment:
should this be static?

Reply:
I think so, because they are never overridden and they don't need an object.